tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -39,6 +39,8 @@ sub run_test {
         record_info "$guest", "ISOLATED NETWORK for $guest";
         ensure_online $_, skip_network => 1;
 
+        check_guest_health($guest);
```

Comment:
Quick question, would you mind adding helpful info here. I still totally no clear why add check_guest_health($guest); before ISOLATED NETWORK test. thanks. is it the debug way for SRIOV failure, right?

Reply:
because we'd like to know if guest is good as soon as possible. Checking just after guest installation is a best place, but in legacy prj1 test, we are unable to get the guest's IP. The first place to get the guest IP is vitual network tests, so it is added here. It is not the best place, but I cannot find a better place.